README.md

```diff
@@ -75,6 +75,44 @@ Salestation allows and recommends you to define your own custom errors. This is
   })
 ```
 
+### Using Extractors
+
+Salestation provides extractors to fetch parameters from the request and pass them to the chain.
+Available extractors are `BodyParamExtractor`, `QueryParamExtractor`, `ConstantInput`.
```

Comment:
Document also that the extractors can be merged and what the merge semantics are (next extractor overwrites 
previous value if same key).